### ISSUE TITLE
Training with images without persons

### ISSUE THREAD

**Issue by tangfayuan** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#145]
The data set is too large and training is slow. If you delete the pictures that do not contain people in the training set, can you still train the key point detection of the human body?

**Comment by Daniil-Osokin** [comment]
Yes, but you have to handle it during training data preparation. You can limit the dataset size by modifying [\_\_len__](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/blob/master/datasets/coco.py#L70) method. Just return 10000 or number you prefer. Although this will lower final accuracy, but it may be enough for your task.

**Comment by tangfayuan** [comment]
Very useful, thank you very much. I have another quertion. During the training process, will the training be conducted on pictures without human beings?  In the 2017train data set, are pictures without people not involved in training?
